lib/src/main/java/graphql/nadel/engine/transform/hydration/NadelHydrationTransform.kt

```diff
@@ -213,6 +213,9 @@ internal class NadelHydrationTransform(
             )
         }
 
+        // This isn't really right… but we start with this
```

Comment:
it would be great to have better details of current issues/limitation

Reply:
Ah sorry yeah I forgot we discussed this, will fix in subsequent PR